app/views/omni_nav/omni_nav_v2.html.erb

```diff
@@ -68,7 +68,12 @@
       </div>
     </div>
     <div class="nav-container primary-nav-search">
-      <%= render 'omni_nav/search_box' %>
```

Comment:
Oh sorry. I didn't realize this was being used in two spots. I think that does justify a partial. My bad.